src/main/java/com/github/onsdigital/babbage/request/handler/list/DataListRequestHandler.java

```diff
@@ -44,23 +46,26 @@ public BabbageResponse get(String uri, HttpServletRequest request) throws IOExce
             return rssService.getDataListFeedResponse(request);
         } else {
             try {
-                return listPage(REQUEST_TYPE, queries(request, extractPublishDates(request), uri));
+                return SearchRendering.buildPageResponse(REQUEST_TYPE, searchAll(queries(request, extractPublishDates(request), uri)));
             } catch (PublishDatesException pEx) {
-                return listPageWithValidationErrors(REQUEST_TYPE, queries(request, publishedAnyTime(), uri),
-                        pEx.getErrors());
+                return SearchRendering.buildPageResponseWithValidationErrors(
```

Comment:
Is this an error we would want to know about? If so we should add a log event here.